### PR TITLE
Travis CI: Initial draft for CI builds.

### PR DESCRIPTION
**Pull request by sunweaver**

### CODE REVIEW

**Comment by sunweaver** [comment]
@tari01: this project is ready for review and merge. Please, take a look.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+On GNU/Linux, Ayatana Indicators are currently available for desktop
+envinronments like MATE (used by default in [Ubuntu
+MATE](https://ubuntu-mate.com)), XFCE (used by default in
+[Xubuntu](https://bluesabre.org/2021/02/25/xubuntu-21-04-progress-update/),
+LXDE, and the Budgie Desktop.
+
+The Lomiri Operating Environment (UI of the Ubuntu Touch OS, formerly
+known as Unity8) uses Ayatana Indicators for rendering its notification
+area and the [UBports](https://ubports.com) project is a core contributor
+to the Ayatana Indicators project.p
```
Comment (tari01) [inline README.md:30]: project.p

File: README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+For further info, please visit:
+https://ayatana-indicators.org
+
+## About this Software Component
+
+This shared library contains functions that two or more system indicator
+components share.
+
+This shared library originally was created to reduce code deduplications
```
Comment (tari01) [inline README.md:40]: was originally
deduplications -> duplication

**Review by tari01 — CHANGES_REQUESTED** [review]
I like it. Small lingo/typo errors pointed out.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+The Ayatana Indicators project is the continuation of Application
+Indicators and System Indicators, two technologies developed by Canonical
+Ltd. for the Unity7 desktop.
+
+Application Indicators are a GTK implementation of the StatusNotifierItem
+Specification (SNI) that was originally submitted to freedesktop.org by
+KDE.
+
+System Indicators are an extensions to the Application Indicators idea.
+System Indicators allow for far more widgets to be display in the
```
[the remark this answers is not in the document]
Comment (sunweaver) [reply README.md]: Amended.

File: README.md (hunk 2 above)
Comment (sunweaver) [reply README.md]: Amended.

File: README.md (hunk 1 above)
Comment (sunweaver) [reply README.md]: Amended.

**Review by tari01 — APPROVED** [review]

**Pull request merged**